apps/earn-protocol-institutions/features/admin/AdminPanelFeedbackList.module.css

```
@@ -41,71 +182,114 @@
-  padding: 10px;
-  border-radius: 10px;
+  margin: 0;
+  padding: 12px;
+  border-radius: 8px;
+  background: rgba(0, 0, 0, 0.2);
+  border: 1px solid rgba(155, 155, 155, 0.1);
   white-space: pre-wrap;
-  font-family: inherit;
-  color: #e5e7eb;
+  word-wrap: break-word;
+  font-family: 'Menlo', 'Monaco', 'Courier New', monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)